hcc_config/hcc_config.hxx.in

```diff
@@ -3,11 +3,6 @@
 #define CMAKE_BUILD_LIB_DIR     "@PROJECT_BINARY_DIR@/lib"
 #define CMAKE_BUILD_COMPILER_RT_LIB_DIR   "@PROJECT_BINARY_DIR@/compiler/lib/clang/5.0.0/lib/linux"
 
-#define CMAKE_INSTALL_PREFIX    "@CMAKE_INSTALL_PREFIX@"
-#define CMAKE_INSTALL_INC           CMAKE_INSTALL_PREFIX"/include"
-#define CMAKE_INSTALL_LIB           CMAKE_INSTALL_PREFIX"/@CMAKE_INSTALL_LIBDIR@"
-
-#define CMAKE_HSA_ROOT         "@HSA_HEADER@"
-#define CMAKE_HSA_LIB          "@HSA_ROOT@/lib"
+#define CMAKE_INSTALL_LIB       "@CMAKE_INSTALL_LIBDIR@"
```

Comment:
That's because the naming convention for library directory varies across different distros.  On Ubuntu it's lib but on Fedora it's lib64.
For include files, it's the same.  I could replace it with a macro if that's more desirable.